.env.example

```diff
@@ -3,3 +3,7 @@ L2_WEB3_ENDPOINTS=
 
 # wallet private key
 PRIVATE_KEY=
+
+STAGE=production
```

Comment:
I think we can remove line 7 and 8 since we set it in index.ts. Just to reduce some confusion.

Reply:
fixed